### PR TITLE
Newsletter PR (7/6)

### PR DESCRIPTION
**Pull request by vs77bb**
@mbeacom @SaptakS @owocki can you test and merge tomorrow, when time permits?

Cc @bsparango

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -471,43 +471,43 @@ def render_start_work_applicant_expired(interest, bounty):
...
+In addition to that, we highlighted <a href="https://medium.com/gitcoin/gitcoin-testimonials-augur-9bfe97368a30">
+Augur's work on the Gitcoin platform.</a>
+In this post, we briefly discuss what the goal of the Augur project is, how the team has used Gitcoin
+to build it, and some specifics about the important bounties they’ve had completed by the Gitcoin community.
+Special shout out to Tom Kysar and @cryptomental for their input on this one!
+</p>
+
+<p>
+Finally, we pushed our
+<a href="https://medium.com/p/804c18dc91da/edit">second installment of Gitcoiner profiles.</a>
```
Comment (owocki) [inline app/retail/emails.py]: edit link - should be public link

Comment (owocki) [reply app/retail/emails.py]: needs image http://bits.owocki.com/0L0Q0n2w3T0v/Screen%20Shot%202018-07-05%20at%206.31.07%20PM.png

**Comment by owocki** [comment]
a few small things to be resolved.. hopefully u can update while were sleeping and we can ship first thing tomo

**Comment by mbeacom** [comment]
The weekly roundup is currently being delivered. @vs77bb

**Pull request merged**